src/main/ApplicationImpl.cpp

```diff
@@ -241,10 +241,6 @@ ApplicationImpl::start()
         mConfig.FORCE_SCP = true;
     }
 
-    if (mConfig.NETWORK_PASSPHRASE.empty())
```

Comment:
I would rather keep checks in one place (ie: here) than move them into Create

Reply:
It was incorrect with the checks here. Application::start is not called when the command line argument --catchupComplete is passed, so these checks could be bypassed in that case. I chose to move it so the checks could not be bypassed in any possibility, even in newly written code. I could move it from create to some other place where it cannot be bypassed if you have any preferences?

Reply:
Didn't realize the code was that messy. We have so many "main loops" in the code :-/ . Leave it invoked at create time then, it seems to be the right place. as it's a few lines of code it can stay in create (we'll refactor into its own method if it gets too big later)